packages/vanilla/src/cells/NumberCell.tsx

```diff
@@ -36,11 +36,13 @@ import { withVanillaCellProps } from '../util/index';
 export const NumberCell = (props: CellProps & VanillaRendererProps) => {
   const { data, className, id, enabled, uischema, path, handleChange } = props;
 
+  const value = data !== undefined ? data : '';
+
   return (
     <input
       type='number'
       step='0.1'
-      value={data || ''}
+      value={value}
```

Comment:
```suggestion
      value={data ?? ''}
```

Reply:
True, thanks! I am making the change now.

Reply:
Also I would love to add a testcase, but for the moment I am not familiar with how enzyme works, or testing for react components in general, so it might take me some time :sweat_smile: